### ISSUE TITLE
Component metadata: Series level

### ISSUE THREAD

**Issue by ggeisler** [sul-dlss/vt-arclight#214]
EDIT: As discussed during standup, this work might not need to be done explicitly for Series if we do the work in #217 and use it for all components. So #217 should be done before determining if any special work is need for Series to get the desired output described below.

For a series-level component show page:

<img width="1009" alt="Screen Shot 2022-11-14 at 2 30 18 PM" src="https://user-images.githubusercontent.com/101482/201769874-a0f8d979-1461-452f-b832-6252a43803c7.png">

---

Stick with what we're currently displaying:

**A: Scope and Contents**
- Continue to display as we are now: All `<scopecontent>` `<p>` values (however many there might be)
- These are currently being written, I believe, and hopefully will be shorter than the one we're currently displaying.
- If we use the `<scopecontent>` `<head>` value as the field label, is it possible to lower-case the "C" in "Contents" so we are using sentence-case, as shown in the mockup?

**B: Acquisition information**
- Continue to display as we are now.
- Since this is the same for every item in the collection, I'm tempted to omit it. But it also seems important to make it easy for the user to understand who the original source of this content is, so I suggest we leave it for now at least.

I'm going to make separate tickets for each level of the inventory, but if it helps, my suggestion is that "Scope and contents" and "Acquisition information" are always the first two elements displayed on every level component page, when they exist. I think, however, that we might only have "Scope and contents" at the series level, which means at all other levels we will start with the "Acquisition information" as the first one we display.

**Comment by ggeisler** [comment]
Closing this in favor of the more general approach in #217.
